### PR TITLE
final touch's before production

### PR DESCRIPTION
**Pull request by MohamedRoshdy2021**
[in](url) this PR i have 

- Implemented Redux createAsyncThunk to fetch data from API and display details. 🟢 
- Developed the application based on the Behance design template for consistency and style. 🟢 
- Utilized Redux Toolkit to create a robust store for state management. 🟢 
- Created a home page to showcase general information for each country. 🟢 
- Designed a detail page to display more detailed information for each country. 🟢  
- Ensured code quality by resolving all linter errors. 🟢 
- Wrote a descriptive README. 🟢 
- the Live link: [Live-Link](https://styling--bright-cheesecake-408665.netlify.app/) 
- the video-Explanation link : [loom-vid](https://www.loom.com/share/6fc2d162e2f14702bfe8584ac5a13cfe) 🟢

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,161 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microvesre who helped us during our journey
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (codepantha) [inline README.md]: - [ ] Please ensure that the "Acknowledgements" section gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @MohamedRoshdy2021 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

- [ ] Please see the requirements below pertaining to the deployment of your application and also recording a demo video 👇 The link to the deployed project should be in the README 🙏 
image

- [ ] You have not met the testing requirements for your this project. Please add unit and integration tests to properly test your project 👍 
image



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please try to follow best practices for your commit messages by ensuring that your commit messages are in the imperative tense. For instance, instead of "Added images to the application", we should have ""Add images to the application."

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — COMMENTED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @MohamedRoshdy2021, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Highlights
-  Implemented required changes. :heavy_check_mark: 
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 



Good luck! 🎉

### Optional Suggestions

- [OPTIONAL] Always ensure you PR title is more descriptive of the project you're working on. The present PR title is not descriptive enough. something like `React Capstone Project` would have been better. 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @MohamedRoshdy2021, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Highlights
-  Implemented required changes. :heavy_check_mark: 
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 



Good luck! 🎉

### Optional Suggestions

- [OPTIONAL] Always ensure you PR title is more descriptive of the project you're working on. The present PR title is not descriptive enough. something like `React Capstone Project` would have been better. 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**